servers/src/main/java/tachyon/master/next/filesystem/FileSystemMaster.java

```diff
@@ -256,11 +257,13 @@ public long getFileId(TachyonURI path) throws InvalidPathException {
     }
   }
 
-  public FileInfo getFileInfo(long fileId) throws FileDoesNotExistException {
+  public FileInfo getFileInfo(long fileId) throws FileDoesNotExistException, InvalidPathException {
     // TODO: metrics
     synchronized (mInodeTree) {
       Inode inode = mInodeTree.getInodeById(fileId);
-      return inode.generateClientFileInfo(mInodeTree.getPath(inode).toString());
+      FileInfo fileInfo = inode.generateClientFileInfo(mInodeTree.getPath(inode).toString());
+      fileInfo.inMemoryPercentage = getInMemoryPercentage(mInodeTree.getPath(inode));
```

Comment:
Yes, good catch. getInMemoryPercentage() is also used by isFullyInMemory, which is used externally